packages/aws-cdk-lib/aws-ecs/lib/base/base-service.ts

```diff
@@ -227,6 +227,32 @@ export interface ServiceConnectService {
    * @default - none
    */
   readonly ingressPortOverride?: number;
+
+  /**
+   * The amount of time in seconds a connection for Service Connect will stay active while idle.
+   *
+   * A value of 0 can be set to disable `idleTimeout`.
+   *
+   * If `idleTimeout` is set to a time that is less than `perRequestTimeout`, the connection will close
+   * when the `idleTimeout` is reached and not the `perRequestTimeout`.
+   *
+   * @default - Duration.minutes(5) for HTTP/HTTP2/GRPC, Duration.hours(1) for TCP.
+   */
+  readonly idleTimeout?: Duration;
+
+  /**
+   * The amount of time waiting for the upstream to respond with a complete response per request for
+   * Service Connect.
+   *
+   * A value of 0 can be set to disable `perRequestTimeout`.
+   * Can only be set when the appProtocol for application container is HTTP/HTTP2/GRPC.
+   *
+   * If `idleTimeout` is set to a time that is less than `perRequestTimeout`, the connection will close
+   * when the `idleTimeout` is reached and not the `perRequestTimeout`.
```

Comment:
Since CloudFormation is not in error, I didn't validate whether `idleTimeout` is less than `perRequestTimeout`. (We shouldn't make errors without a lot of thought, and I didn't think it was enough to use annotations.)

I also thought about replacing the `perRequestTimeout` value with an `idleTimeout`, but I wasn't sure if that would suit the user's intentions, so I did nothing for now.